Andromeda.Commands/RunJobsCommand.cs

```diff
@@ -7,16 +7,18 @@
 using Jobs.Fetcher.YouTube;
 using Jobs.Fetcher.Facebook;
 using Jobs.Fetcher.Twitter;
+using Jobs.Fetcher.TikTok;
 
 namespace Andromeda.Commands {
 
     public static class RunJobsCommand {
 
         public static JobsFactory[] Factories = new JobsFactory[] {
             new YouTubeFetchers(),
-            new AdWordsFetchers(),
```

Comment:
Maybe disabling Adwords and reenabling Twitter should go in a separate PR -- but I'm just being pedantic

Reply:
I agree it should, but I had already created so many different PRs that I decided to leave this one here